src/python/zquantum/qcbm/ansatz.py

```diff
@@ -19,7 +19,11 @@ class QCBMAnsatz(Ansatz):
     topology = ansatz_property("topology")
 
     def __init__(
-        self, number_of_layers: int, number_of_qubits: int, topology: str = "all",
+        self,
+        number_of_layers: int,
+        number_of_qubits: int,
+        topology: str = "all",
+        **topology_kwargs
```

Comment:
Looks good to me, @mstechly. If using `topology_kwargs` instead of `kwargs` raised your concerns – don't worry! The latter is just a conventional name for cases when you don't have a better idea for the variable that stores additional keyword arguments :) I'd say that it's actually a good practice to add a prefix to hint what kind of keywords might be expected.